stable/spinnaker/values.yaml

```diff
@@ -40,6 +40,10 @@ halyard:
   ## run by the init container before halyard is started.
   # additionalInitScript: |-
 
+  ## Uncomment if you want to add annotations on halyard and install-using-hal pods
+  #annotations:
```

Comment:
@costimuraru CircleCI says `missing starting space in comment`.
This should be `# annotations:`

Reply:
Thanks @hareku, added the space.